data/json/items/magazine/shot.json

```diff
@@ -1,5 +1,15 @@
 [
   {
+    "id": "beltshot",
+    "copy-from": "magazine_belt",
+    "type": "MAGAZINE",
+    "name": "12 gauge ammo belt",
+    "ammo_type": "shot",
+    "capacity": 100,
+    "count": 100,
+    "linkage": "ammolinkshot"
```

Comment:
RL-wise, linkages are particular to a single caliber of bullet- (they need to fit snugly around the casing so they stay properly linked.

For something like our abzats shotgun, I'd go with an old-fashioned cloth ammo belt due to its ease of construction and reusability compared to disintegrating linkages.